lib/runtime/storage/trie_test.go

```diff
@@ -40,10 +40,56 @@ func TestTrieState_SetGet(t *testing.T) {
 	testFunc(ts)
 }
 
+func TestTrieState_SetGetChildStorage(t *testing.T) {
+	testFunc := func(ts *TrieState) {
+		for _, tc := range testCases {
+			childTrie := trie.NewEmptyTrie()
+			err := ts.SetChild([]byte(tc), childTrie, trie.V0)
+			require.NoError(t, err)
+
+			err = ts.SetChildStorage([]byte(tc), []byte(tc), []byte(tc), trie.V0)
+			require.NoError(t, err)
+		}
+
+		for _, tc := range testCases {
+			res, err := ts.GetChildStorage([]byte(tc), []byte(tc))
+			require.NoError(t, err)
+			require.Equal(t, []byte(tc), res)
+		}
+	}
+
+	ts := &TrieState{t: trie.NewEmptyTrie()}
+	testFunc(ts)
+}
```

Comment:
What is the reason of having `testFunc`? Do we really need this extra layer of function?

Reply:
Agree, I removed it